ci/builders/linux_fuchsia.json

```diff
@@ -164,6 +164,7 @@
                 "parameters": [
                     "--engine-version",
                     "${REVISION}",
```

Comment:
The "Magic variables" section of the docs here https://github.com/flutter/engine/blob/main/ci/builders/README.md#magic-variables should explain that `${REVISION}` is the empty string in presubmit.

Reply:
You are correct, revision is empty for presubmit. I'll update the docs.